code/game/machinery/kitchen/cooking_machines/candy.dm

```diff
@@ -1,6 +1,6 @@
 /obj/machinery/appliance/mixer/candy
 	name = "candy machine"
-	desc = "Get yer candied cheese wheels here!"
```

Comment:
I suggest keeping this as a fluff desc.

Reply:
nah i don't think that's worth a desc_fluff